src/components/Tags/MultiFilterStyleTagSelector.tsx

```
@@ -483,164 +497,135 @@ export function MultiFilterStyleTagSelector({
+                              {isLoadingChildren ? (
+                                <div className="p-2 text-sm text-gray-500">
+                                  {t("loading")}
+                                </div>
+                              ) : childTags?.results?.length ? (
+                                childTags.results.map((childTag) => (
+                                  <div
+                                    key={childTag.id}
+                                    className="focus:bg-gray-100 focus:text-gray-900 relative rounded-sm text-sm outline-hidden select-none flex items-center gap-2 px-2 py-1 cursor-pointer"
+                                    onClick={() => handleSelect(childTag)}
+                                  >
+                                    <button
```

Biome (2.3.14)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)